app/components/AddDelegateModal/index.js

```diff
@@ -81,15 +56,35 @@ class AddDelegateModal extends Component<Props> {
     );
   };
 
-  renderCreationBody = () => {
+  createAccount = async () =>  {
+    const { createNewAccount, selectedParentHash, onCloseClick } = this.props;
     const { delegate, amount, fee, passPhrase } = this.state;
-    const isDisabled = !delegate || !amount || !fee || !passPhrase;
+    this.setIsLoading(true);
+    if ( await createNewAccount( delegate, amount, fee, passPhrase, selectedParentHash ) ) {
+      this.setState(defaultState);
+      onCloseClick();
+    } else {
+      this.setIsLoading(false);
+    }
+  };
+
+  render() {
+    const { open, onCloseClick } = this.props;
+    const { isLoading, delegate, amount, fee, passPhrase } = this.state;
+    const isDisabled = isLoading || !delegate || !amount || !fee || !passPhrase;
```

Comment:
Will it ever be possible that `amount` or `fee` could be `0`? I'm guessing not, but if so they will register as falsey

Reply:
@vishakh Is there a ticket for setting up the custom fee? Right now it's just set at 500